preview-src/index.ts

```diff
@@ -3,9 +3,11 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 import './index.css';
-import { renderTimelineEvent, getStatus, renderComment, PullRequestStateEnum, renderReview, TimelineEvent, EventType } from './pullRequestOverviewRenderer';
-import md from './mdRenderer';
 import * as moment from 'moment';
+import { TimelineEvent, EventType } from '../src/common/timelineEvent';
+import { PullRequestStateEnum, IAccount } from '../src/github/interface';
```

Comment:
We have two different tslint config files, one in this directory and one at the root. The one here has `strict: true` and `noImplicitAny: true` which the root level one doesn't. I'm seeing some linter errors from these for this imported file when building. Let's remove those compiler options temporarily so this PR stays focused